contracts/TokenCrowdsale.sol

```diff
@@ -54,16 +54,15 @@ contract TokenCrowdsale is MintedCrowdsale, FinalizableCrowdsale {
      require(_uncappedOpeningTime <= _closingTime);
      require(_uncappedOpeningTime >= _openingTime);
 
-     require(_icoTotalCap > 0);
-
      rates = _rates;
      capsTo = _capsTo;
      uncappedOpeningTime = _uncappedOpeningTime;
-     icoTotalCap = _icoTotalCap;
+     icoTotalCap = _token.ICO_SALE_CAP();
```

Comment:
no need to save this separately - you can always access it from toke

Reply:
Fixed.